src/runtime/components/nuxt-img.ts

```diff
@@ -94,6 +94,9 @@ export default defineComponent({
                 imagesizes: sizes.value.sizes,
                 imagesrcset: sizes.value.srcset,
               }),
+          ...(typeof props.preload !== 'boolean' && props.preload.fetchPriority
+            ? { fetchpriority: props.preload.fetchPriority }
+            : {}),
```

Comment:
should we default to `fetchpriority: 'high'`?

Reply:
Sorry for not answering in time, this comment fell off my radar 🌠

Reply:
Still time to update - what do you think?

Reply:
Why should the fetch priority default to high in your opinion? I'd say we can just leave it for the users to decide if they want to specify any.